app/__init__.py

```diff
@@ -3,31 +3,32 @@
 from flask import Flask, request, redirect
 from flask_login import LoginManager
 from flask._compat import string_types
-from dmutils import apiclient, logging, config, proxy_fix
+from dmutils import apiclient, init_app, flask_featureflags
 
 from config import configs
 from .model import User
 
 data_api_client = apiclient.DataAPIClient()
 login_manager = LoginManager()
+feature_flags = flask_featureflags.FeatureFlag()
 
 
 def create_app(config_name):
     application = Flask(__name__,
                         static_folder='static/',
                         static_url_path=configs[config_name].STATIC_URL_PATH)
-    application.config.from_object(configs[config_name])
-    configs[config_name].init_app(application)
-    config.init_app(application)
+
+    init_app(
+        application,
+        configs[config_name],
+        data_api_client=data_api_client,
+        feature_flags=feature_flags,
+        login_manager=login_manager,
+    )
 
     from .main import main as main_blueprint
     from .status import status as status_blueprint
 
-    proxy_fix.init_app(application)
```

Comment:
It's being done in [digitalmarketplace-utils](https://github.com/alphagov/digitalmarketplace-utils/blob/master/dmutils/__init__.py#L25) since this past Friday.